src/main/windows/widget.ts

```diff
@@ -2,18 +2,20 @@ import { BrowserWindow, screen } from 'electron';
 
 import { isAutoLaunchEnabled } from '../auto-launch/service';
 import eventBus from '../event-bus';
-import { getTray } from '../tray';
+import { TrayMenu } from '../tray';
 import { preloadPath, resolveHtmlPath } from '../util';
 import { setUpCommonWindowHandlers } from '.';
 import { getIsLoggedIn } from '../auth/handlers';
 
+const widgetConfig: { width: number; height: number; placeUnderTray: boolean } =
```

Comment:
Do we want to keep the window config like before if this window will only display the widget? We could simplify some logic. Will add a comment of what I mean